source/devices/AM335X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -309,8 +309,7 @@ Composer Studio.
 .. rubric:: Host Support
    :name: host-support
 
-The Processor SDK is developed, built and verified on Ubuntu 22.04. Details on how to create a virtual machine to load Ubuntu
-are described in :ref:`this page <how-to-build-a-ubuntu-linux-host-under-vmware>`.
```

Comment:
Well the section "How to Build a Ubuntu Linux host under VMware" [1] is no longer maintained & carries stale information.
 
IMO, we should delete the entire "How_to_Build_a_Ubuntu_Linux_host_under_VMware" file & remove references of it from all the <SOC>_linux_toc tree as well

[1]: https://software-dl.ti.com/processor-sdk-linux/esd/AM65X/09_01_00_01/exports/docs/linux/How_to_Guides/Host/How_to_Build_a_Ubuntu_Linux_host_under_VMware.html

Reply:
This file has been used for devices other than those in the SITARA platform. Currently, its references have been removed from all the _linux_toc tree for SITARA devices